lib/developer_portal/lib/cms/toolbar.rb

```diff
@@ -113,7 +113,9 @@ def cms_toolbar_with_iframe_html
   end
 
   class View < ActionView::Base
-    include ::Rails.application.routes.url_helpers
-    include ::DeveloperPortal::CMS::ToolbarHelper
+    # FIXME: There are shared routes between both
```

Comment:
@3scale/system-ruby this is something we should never do... there are paths that exist in main app and in the dev portal.

Reply:
Oh I am totally mistaken here!!!!! Sorry I will fix